stories/Resources.js

```diff
@@ -4,6 +4,14 @@ import { storiesOf } from '@storybook/react'
 import { DragableCalendar } from './helpers'
 import resources from './helpers/resourceEvents'
 
-storiesOf('Resources').add('demo', () => (
-  <DragableCalendar events={resources.events} resources={resources.list} />
-))
+storiesOf('Resources')
```

Comment:
The repository seems doesn't have component tests e.g. enzyme + snapshot.

What should I do to test my changes on components?